x-pack/test/api_integration/services/secops_graphql_client.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { format as formatUrl } from 'url';
+import fetch from 'node-fetch';
+import { InMemoryCache, IntrospectionFragmentMatcher } from 'apollo-cache-inmemory';
+import { ApolloClient } from 'apollo-client';
+import { HttpLink } from 'apollo-link-http';
+
+import introspectionQueryResultData from '../../../plugins/secops/common/graphql/introspection.json';
+
+export function SecOpsGraphQLProvider({ getService }) {
+  const config = getService('config');
+  const kbnURL = formatUrl(config.get('servers.kibana'));
+
+  return new ApolloClient({
+    cache: new InMemoryCache({
+      fragmentMatcher: new IntrospectionFragmentMatcher({
+        introspectionQueryResultData,
+      }),
+    }),
+    link: new HttpLink({
+      credentials: 'same-origin',
+      fetch,
+      headers: {
+        'kbn-xsrf': 'xxx',
```

Comment:
what does passing the value `xxx` do here?

Reply:
After researching on Elastic forum, I can tell you that we need it because Kibana expects the header 'kbn-version' to be passed on requests, it's used to protect against xsrf attacks.

Reply:
Thanks, in that case, would a `TODO` and/or a github issue make sense here?